Resources/public/js/views/ez-confirmboxview.js

```diff
@@ -80,6 +83,45 @@ YUI.add('ez-confirmboxview', function (Y) {
         },
 
         /**
+         * Hides of reveal the buttons
+         *
+         * @method _uiButtonDisplay
+         * @protected
+         */
+        _uiUpdateButton: function () {
+            var container = this.get('container').one('.ez-confirmbox-tools');
+
+            if (this.get('renderDefaultButtons')) {
+                container.removeClass('ez-confirmbox-tools-hidden');
+            } else {
+                container.addClass('ez-confirmbox-tools-hidden');
+            }
+        },
+
+        /**
+         * Updates the view in the rendered confirmBox
+         *
+         * @method _uiUpdateView
+         * @protected
+         */
+        _uiUpdateView: function () {
+            var view = this.get('view');
+            if (view instanceof Y.View) {
```

Comment:
You should add an empty line after variable declaration (and maybe also delete the empty line after the 'if')